conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/mantis/EthereumFilterFromQueryString.scala

```diff
@@ -0,0 +1,39 @@
+package tech.cryptonomic.conseil.api.routes.platform.data.mantis
+
+import cats.syntax.functor._
+import endpoints.algebra.JsonEntities
+import tech.cryptonomic.conseil.api.routes.platform.data.ApiFilter.Sorting
+import tech.cryptonomic.conseil.api.routes.platform.data.ApiFilterQueryString
+import tech.cryptonomic.conseil.common.util.TupleFlattenUtil.FlattenHigh._
+import tech.cryptonomic.conseil.common.util.TupleFlattenUtil._
+
+private[mantis] trait MantisFilterFromQueryString extends ApiFilterQueryString { self: JsonEntities =>
```

Comment:
mismatch between`trait `MantisFilterFromQueryString` and filename: `EthereumFilterFromQueryString`